docs/dev/model/User.md

```diff
@@ -0,0 +1,28 @@
+# User
+
+A User of the PASS system. This includes prefered person information that can be used to autopopulate [Contributor](Contributor.md) records
+
+| Field         | Type          | Description |
+| ------------- | ------------- | ------------- |
+| id* | String | Autogenerated identifier of object |
+| username* | String | Unique login name used by User |
+| firstName | String | First name(s) of User |
+| middleName | String | Middle name(s) of User |
+| lastName | String | Last name(s) of User |
+| displayName | String | Name for display. Separate names may not be available, but a person should always at least have a display name. |
+| email | String | Contact email for User |
+| affiliation | String[] | The affiliation(s) of the User with their institution, for example `STAFF@inst.edu`.  An institution may have multiple organizational units, and a User may have a different affiliation with any given OU.  A User having an affiliation with multiple OUs in an institution would have multiple values, for example `FACULTY@medicine.inst.edu` and `STUDENT@engineering.inst.edu`. |
```

Comment:
Which table should affiliation be? I'm looking in staging at the moment and I don't see anything like `pass_user_affiliation`.  Perhaps it should be a String and not a String array? I see for locatorIds[], there is a table for that relationship: `pass_user_locators`